core/src/apps/ton/sign_message.py

```
@@ -0,0 +1,155 @@
+from apps.ton.tonsdk.utils._address import Address
+
+from . import ICON, PRIMARY_COLOR, tokens
+from .layout import require_confirm_fee, require_show_overview
+
+if TYPE_CHECKING:
+    from trezor.wire import Context
+
+
+@auto_keychain(__name__)
+async def sign_message(
+    ctx: Context, msg: TonSignMessage, keychain: Keychain
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)